src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/model/induction/CiagInductionResponseAssert.kt

```diff
@@ -19,16 +19,6 @@ class CiagInductionResponseAssert(actual: CiagInductionResponse?) :
     CiagInductionResponseAssert::class.java,
   ) {
 
-  fun hasAReference(): CiagInductionResponseAssert {
```

Comment:
As per my comment in `GetInductionTest`, these tests were redundant, since the fields in question can never be null.

Reply:
Gotcha 👍 😁